docs/API_reference.md

```diff
@@ -275,6 +276,23 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+---
+
+##### plenty_api_get_property_selections: <a name='get-property-selections'></a>
+
+Return a mapping of Selection values for selction properties.
+
+[*Optional parameter*]:
+
+The **refine** field can be used to reduce the request to specific property IDs.
+example: refine={'propertyID': 123}
+
+[*Output format*]:
+
+There are currently two supported output formats: 'json' and 'dataframe'.  
+The 'json' format returns a dictionary containing the property_id, selection_id, language and finally the value.
```

Comment:
I find this description of the dictionary a bit vague because this sounds like:
{'property_id': 123, 'selection_id': 12, 'language': 'de', 'value': 'value_1'}
but it is:
`{123: {12: {'de': 'value_1'}}}`
So maybe a better description is:
... returns a dictionary of property IDs to its available selection values

And maybe add a small sample result, so that the user knows what to expect.